app/styles/globals.scss

```diff
@@ -27,13 +28,34 @@ body {
 }
 
 .page-container {
-  max-width: 600px;
+  max-width: 1000px;
 }
 
 .container-data {
   background: var(--background-tint);
 }
 
+.doses-data-grid {
+  display: grid;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+  gap: 1.25rem;
```

Comment:
@finnclark - this is a gap! Will use one of your space vars after your PR is merged